### DISCUSSION TITLE
Welcome to aqara-mqtt Discussions!

### DISCUSSION THREAD

**Discussion by monster1025** [monster1025/aqara-mqtt#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kiran5k4u** [comment]
I ran this script in python and there was the following error. do you know what the problem can be?

--------------
INFO:__main__:Init mqtt client.
INFO:mqtt:Connecting to MQTT server 192.168.1.110:1883 with username (:)
INFO:mqtt:Starting mqtt loop.
INFO:mqtt:Subscribing to home/gateway/+/+/set.
INFO:mqtt:Connected to mqtt server.
INFO:mqtt:Subscribing to home/gateway/+/write.
INFO:mqtt:Subscribing to home/plug/+/status/set.
INFO:xiaomihub:"Read unwanted data" fix is False
INFO:xiaomihub:Discovering Xiaomi Gateways using address 192.168.1.131
INFO:xiaomihub:Gateway found on IP 192.168.1.131
INFO:xiaomihub:Creating Multicast Socket
File "C:\Users\Xiaomi\src\main.py", line 114, in <module>
    gateway = XiaomiHub(gateway_pass, gateway_ip, config)

  File "C:\Users\Xiaomi\src\xiaomihub.py", line 72, in __init__
    self._mcastsocket = self._create_mcast_socket()

  File "C:\Users\xiaomihub.py", line 182, in _create_mcast_socket
    sock.bind((self.MULTICAST_ADDRESS, self.MULTICAST_PORT))

OSError: [WinError 10049] The requested address is not valid in its context
--------------
apprently it do not like the  MULTICAST_ADDRESS = '224.0.0.50'.

Any thhoughts?

Best regards
Kiran Munji
